.gitlab/pnnl/README.md

```diff
@@ -0,0 +1,86 @@
+# How to connect GitHub to a PNNL GitLab (push) Mirror
+
+To run on our HPC clusters at PNNL while hosting our code base on GitHub, we utilize PNNL's CI/CD services on GitLab. 
+
+With GitLab premium, integration between GitHub and GitLab is included. This is how we have architected a solution with the base level GitLab offering. 
+
+The PNNL GitLab repository is a push mirror of the GitHub. So whenever a commit is pushed to a pull request or the `main` branch - the changes are pushed to the GitLab and a CI pipeline is triggered.
+
+## Steps in linking the GitLab and GitHub
+
+1. Push mirror & pipeline trigger
+
+In `pnnl_mirror.yaml`, we use the variables `GIT_USER`, `GIT_PASSWORD`, and `PNNL_PIPELINE_TRIGGER`. `GIT_PASSWORD` and `PNNL_PIPELINE_TRIGGER` are generated in GitLab and then added to GitHub.
+
+(TODO: is GIT_USER just something you add directly into github?)
+
+a) GIT_PASSWORD
+Go to Settings > Access Tokens. Click `Add new token`.
+
+Pick a reasonable name & expiration date. (ie "GITHUB_PUSH_PASSWORD")
+
+Pick the `Developer` role.
+
+Under `Select scopes`, select `write_repository`.
+
+Create.
+
+Go to GitHub > Settings > Secrets and variables > Actions. Click `New repository secret`.
+Name it `GIT_PASSWORD` and paste in the value generated from GitLab.
+
+Add secret.
+
+b) PNNL_PIPELINE_TRIGGER
+Go to Settings > CI/CD > Pipeline trigger tokens. Click `Add new token`.
+
+Give it a name and click create. 
+
+Go to GitHub > Settings > Secrets and variables > Actions. Click `New repository secret`.
+Name it `PNNL_PIPELINE_TRIGGER` and paste in the value generated from GitLab.
```

Comment:
Since @jaelynlitz you made the secret, you will show up as running the pipelines. Would be good to offload that to the group somehow like you can with GitHub and their config, but I don't think you can...